crates/nu-command/src/filters/find.rs

```diff
@@ -147,6 +147,8 @@ impl Command for Find {
     ) -> Result<PipelineData, ShellError> {
         let regex = call.get_flag::<String>(engine_state, stack, "regex")?;
 
+        let input = split_string_if_multiline(input);
```

Comment:
Thanks for the quick turnaround! This placement here could interfere with the `--multiline --regex ...` mode, I think.